cypress/integration/TC05_partners.spec.js

```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress" />
+let LOCAL_URL = 'http://localhost:3000/partners';
+
+describe('Partners Flow', () => {
+  beforeEach(() => {
+    cy.visit(LOCAL_URL);
+  });
+  it('partners page is shown correctly', () => {
```

Comment:
we have been formatting cypress tests so individual tests read like "it should show the page content correctly". possibly change this to:

`it('should show page content correctly, () => {'`